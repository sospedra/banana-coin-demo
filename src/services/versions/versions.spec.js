/* global expect, describe, it */
const { createVersion } = require('./')

jest.mock('express', () => ({
  Router: jest.fn(() => ({ use: jest.fn() }))
}))

describe('service versions', () => {
  it('should create a version with a possible callback', async () => {
    const callback = jest.fn()

    expect(createVersion(callback)).toMatchSnapshot()
    expect(callback).toHaveBeenCalled()
  })
})
