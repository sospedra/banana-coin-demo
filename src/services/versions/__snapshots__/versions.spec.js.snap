// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`service versions should create a version with a possible callback 1`] = `
Object {
  "use": [MockFunction] {
    "calls": Array [
      Array [
        [Function],
      ],
    ],
  },
}
`;
